Linux Script/configure-linux.sh

```diff
@@ -458,8 +458,9 @@ checkIfSelinuxServiceEnforced()
 	if [ $? -ne 0 ]; then
 		logMsgToConfigSysLog "INFO" "INFO: selinux status is not enforced."
 	elif [ $(getenforce | grep "Enforcing" | wc -l) -gt 0 ]; then
-		logMsgToConfigSysLog "ERROR" "ERROR: selinux status is 'Enforcing'. Please disable it and start the rsyslog daemon manually."
-		exit 1
+	        logMsgToConfigSysLog "Info" "Info: selinux status is 'Enforcing'. Setting it to the permissive mode and restarting the rsyslog daemon."
+		setenforce 0
+		restartRsyslog
 	fi
```

Comment:
@mchaudhary and @mostlyjason, while running our script on different distributions, I found one more thing that was breaking our script execution(on RedHat) i.e. if **SELinux** status is **enforcing** in the system then the script exits by showing the above error message which I thought should not happen so if **SELinux** status is set to **enforcing** then I am showing an info message and setting the **SELinux** status to **permissive mode** **by setting its value to 0** and **restarting the rsyslog service** so that changes take effect. In this way, our script will not exit in between and will run smoothly.